README.md

```diff
@@ -220,21 +220,34 @@ This plugin creates device plugin endpoints based on the configurations given in
                     "vendors": ["8086"],
                     "devices": ["0d90"]
             }
+        },
+        {
+          "resourceName": "intel_sriov_netdevice_additional_env",
+          "selectors": {
+            "vendors": ["8086"],
+            "devices": ["154c", "10ed", "1889"],
+            "drivers": ["i40evf", "ixgbevf", "iavf"]
+          },
+          "additionalEnvVariables": {
```

Comment:
name is confusing IMO as it does not add any new additional env variable.

I have commented on this in `types.go` so lets discuss it there